rdkcentral/rdkservices
# RDKTV-9258: Set property tts-mode for TTS audio

Reason for change: Amlogic supports tts-mode for
TTS audio
Test Procedure:
1. Turn on voice guidance and navigate through tiles.
while voice guidance is narrating a longer sentence,
execute the following
gst-launch-1.0 httpsrc location="http://localhost:50050/
nuanceEve/tts?voice=ava&language=en-US&rate=50&text=How_are_you"
! audio/x-raw, rate=22050, channels=1, format=S16LE,
layout=interleaved ! audioconvert ! audioresample !
amlhalasink direct-mode=false
Audio from both sources should be heard without any
glitches.
2. No regression for voice guidance

Risks: Low

Signed-off-by: Simi Mathew <simim@tataelxsi.co.in>
